Cleanup Default

larson-carter opened the pull request:
I replaced all of the defaults that were included in VueJS. I also removed some of the non essential parts of code to get ready to make the UI/UX.

Closes: https://github.com/MLH-Fellowship/jobtrack/issues/11

Commit:
Cleanup Default

I replaced all of the defaults that were included in VueJS. I also removed some of the non essential parts of code to get ready to make the UI/UX.

Commit:
Add back About.vue

I accidentally deleted About.Vue

BaffourAdu commented on src/router/index.js:
```diff
@@ -1,6 +1,6 @@
 import Vue from 'vue'
 import VueRouter from 'vue-router'
-import Home from '../views/Home.vue'
```
@larson-carter I would suggest the Component Name matches the file name. Thus, we should use `Home.vue` instead of `HomeView.vue` for our views

BaffourAdu commented on src/router/index.js:
```diff
@@ -9,14 +9,6 @@ const routes = [
     path: '/',
     name: 'Home',
     component: Home
-  },
```
Route for About View missing and I would suggest we lazy-load all routes like this `component: () => import(/* webpackChunkName: "about" */ '../views/About.vue')`

BaffourAdu reviewed (changes requested):
Great work so far! Just a few changes to be made

Commit:
Update Routes + rename Scheme

I updated the routes to support lazy loading.

I also renamed the Views to Just the name

BaffourAdu committed:
Add webpack chunk names

BaffourAdu committed:
Add space after chuck name

BaffourAdu reviewed (approved):
Nice work! @larson-carter

larson-carter commented:
LGTM
